UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.IntegrationTests/RadioNavigationalWarning/Admin/RadioNavigationalWarningAdminTest.cs

```diff
@@ -0,0 +1,174 @@
+﻿using FakeItEasy;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using NUnit.Framework;
+using System;
+using System.Threading.Tasks;
+using UKHO.MaritimeSafetyInformation.Common.Configuration;
+using UKHO.MaritimeSafetyInformation.Common.Models.RadioNavigationalWarning;
+using UKHO.MaritimeSafetyInformation.Web.Controllers;
+using UKHO.MaritimeSafetyInformation.Web.Services;
+using UKHO.MaritimeSafetyInformation.Web.Services.Interfaces;
+
+namespace UKHO.MaritimeSafetyInformation.IntegrationTests.RadioNavigationalWarning.Admin
+{
+    [TestFixture]
+    public class RadioNavigationalWarningAdminTest : BaseRNWTest
+    {
+        public ILogger<RadioNavigationalWarningsAdminController> _fakeLogger;
```

Comment:
```suggestion
        private ILogger<RadioNavigationalWarningsAdminController> _fakeLogger;
```

Reply:
Changes are done as requested, Thank you.